check.js

```diff
@@ -111,7 +112,12 @@ export async function check(
 
     // istanbul ignore next
     if (!error.file) {
-      push(error)
+      let message = error.messageText.messageText;
+      // To support snapshot tests, remove absolute paths that would vary between machines.
+      if (IS_TEST) {
+        message = message.replaceAll(process.cwd(), '$CWD')
```

Comment:
```suggestion
        message = message.replaceAll(process.cwd(), '.')
```

Let’s use `./`

Reply:
Done in https://github.com/ai/check-dts/pull/41/commits/e09d5ce798a403673e366a0fa16feccdc587f21a.